pebbles/static/partials/user_dashboard.html

```diff
@@ -1,28 +1,30 @@
 <div id="user-dashboard">
-    <h1>Blueprints</h1>
+    <h2 style="padding-left: 15px;">Resources</h2>
     <div class="row">
-        <div class="alert alert-warning" ng-hide="blueprints != undefined && (blueprints | filter : {is_enabled: 'true'}).length">
+        <div class="alert alert-warning" style= "margin-left: 30px;" ng-hide="blueprints != undefined && (blueprints | filter : {is_enabled: 'true'}).length">
             No blueprints available
         </div>
         <div ng-repeat="choice in oddEvenRange()" class="col-md-6">
             <div ng-repeat="blueprint in blueprints | filter : { is_enabled: 'true' }" ng-if="filterOddEven($index, choice)" class="col-md-12">
                 <div class="panel panel-default">
                     <div class="panel-heading">
-                        <h3 class="panel-title">{{ blueprint.name }} - {{ blueprint.group_name }}</h3>
+                        <h3 class="panel-title">{{ blueprint.name }} </h3>
```

Comment:
Extra whitespace at the end may make UI automation difficult. Currently it's a substring match but still rather no whitespace.